README.md

```diff
@@ -104,19 +104,26 @@ self-contained environment.
 Prerequisites:
 * A UMLS account
 * A working Docker toolchain, which has been assigned at least 10GB of RAM (The Metathesaurus step requires 8GB of RAM for the Java process)
+  * Note: the Docker terminology process will not run unless Docker has access to at least this much RAM.
```

Comment:
```suggestion
  * Note: the Docker terminology process will not run unless Docker has access to at least this 10GB of RAM.
```

otherwise I could see some folks getting confused about whether 10 or 8 GB is required from the bullet above.